cranelift/frontend/src/ssa.rs

```diff
@@ -162,6 +170,20 @@ impl SSABuilder {
             && self.results.is_empty()
             && self.side_effects.is_empty()
     }
+
+    fn new_epoch(&mut self) -> Epoch {
```

Comment:
Can we have a doc-comment here indicating what a "new epoch" means (amortized visited-bit-clearing) and when we do it?